ks-svarut/src/main/java/no/difi/meldingsutveksling/ks/mapping/edu/FileTypeHandler.java

```diff
@@ -0,0 +1,52 @@
+package no.difi.meldingsutveksling.ks.mapping.edu;
+
+import no.difi.meldingsutveksling.dokumentpakking.service.CmsUtil;
+import no.difi.meldingsutveksling.ks.Dokument;
+import no.difi.meldingsutveksling.ks.mapping.Handler;
+import no.difi.meldingsutveksling.noarkexchange.schema.core.DokumentType;
+
+import javax.activation.DataHandler;
+import javax.mail.util.ByteArrayDataSource;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.security.cert.X509Certificate;
+import java.util.function.Function;
+
+/**
+ * Used to map domain FileTypeHandler to
+ */
+public class FileTypeHandler implements Handler<Dokument.Builder> {
+    private final DokumentType domainDocument;
+    private X509Certificate certificate;
+    private final Function<byte[], byte[]> noop = b -> b;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1170)